build/postgres/Dockerfile

```diff
@@ -33,6 +66,13 @@ RUN if [ "$DFSET" = "centos" ] ; then \
         	psmisc \
         	rsync \
         	wal2json${PG_MAJOR//.} \
+        	file \
+        	gettext \
+        	hostname \
+        	nss_wrapper \
+        	procps-ng \
+        	pgaudit_analyze \
```

Comment:
```suggestion
		pgaudit${PG_MAJOR//.} \
		pgaudit${PG_MAJOR//.}_set_user \
        	pgaudit_analyze \
```

Reply:
`pgaudit${PG_MAJOR//.}` is already added in centos block on this [line](https://github.com/CrunchyData/crunchy-containers/blob/f7236965d8d022306ebddd9b8c15e42bf1aabdd6/build/postgres/Dockerfile#L59). I pulled the centos image and the `pgaudit_analyze` tool was not installed. If that is needed I think we will have to install it separately.

I added `pgaudit_analyze` to the centos block because it was installed specifically in the "not-centos" block for the original `crunchy-postgres` dockerfile.

Reply:
@jmckulk `pgaudit${PG_MAJOR//.}_set_user \` is still missing.